core/src/price_reporter/reporter.rs

```diff
@@ -6,19 +6,20 @@ use serde::{Deserialize, Serialize};
 use stats::median;
 use std::{
     collections::{HashMap, HashSet},
-    fmt::{self, Display},
-    iter::FromIterator,
     num::NonZeroUsize,
     sync::{Arc, RwLock},
 };
 
 use super::{
     errors::ExchangeConnectionError,
-    exchanges::{get_current_time, Exchange, ExchangeConnection, ExchangeConnectionState},
```

Comment:
I'd ignore this file in review, it'll be reimplemented in a follow-up. Commented a lot of things out to check for errors elsewhere.